x-pack/legacy/plugins/lens/public/datatable_visualization_plugin/expression.tsx

```diff
@@ -58,29 +57,32 @@ export const datatable: ExpressionFunction<
       }),
     },
     columns: {
-      types: ['lens_datatable_columns'],
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      types: ['lens_datatable_columns'] as any,
```

Comment:
If i understand correctly this type cast is necessary because the `DatatableColumns` interface used in `Args` does not contain a `type` property. Can we fix the type to make this work? E.g. by defining the args interface like this:
```
interface Args {
  title: string;
  columns: DatatableColumns & { type: 'lens_datatable_columns' };
}
```

Reply:
Done.